iOS/DuckDuckGo/UICoordination/MainCoordinator.swift

```diff
@@ -170,7 +170,7 @@ final class MainCoordinator {
 
     func presentNetworkProtectionStatusSettingsModal() {
         Task {
-            if let hasEntitlement = try? await subscriptionManager.isEnabled(feature: .networkProtection),
+            if let hasEntitlement = try? await subscriptionManager.isFeatureIncludedInSubscription(.networkProtection),
```

Comment:
I think here we have a change in behaviour.
Previously `isEnabled` wa calling `isFeatureAvailableAndEnabled(feature: feature, cachePolicy: .returnCacheDataElseLoad)` that was a merge of Subscription and user entitlements. Now `isFeatureIncludedInSubscription` is only checking the Subscription entitlements.

This is valid for all places where we used `isEnabled(feature`